src/pudl_archiver/archivers/classes.py

```diff
@@ -165,6 +164,42 @@ async def get_hyperlinks(
 
         return hyperlinks
 
+    def validate_archive(
```

Comment:
Eventually we can aggregate generalize-able tests in this method.

Reply:
Meaning we would have some validations that are run for every dataset, and specified here, and concatenated with the list of validation results that comes from the dataset-specific validation method?

Reply:
Yeah that's the idea